hs_core/management/commands/migrate_resources_from_userzone_to_datazone.py

```diff
@@ -0,0 +1,88 @@
+from django.core.management.base import BaseCommand
```

Comment:
how many resources are on the user zone that will be migrated?

Reply:
@sblack-usu See the copy-and-paste output above from test run of this migration command run on beta - 41 resources were migrated from user zone to data zone on beta test run. This number may not be exactly the same as the production migration run depending on when the latest iRODS user zone resource copy snapshot were made, but should be close.